src/HomePage/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { FunctionComponent } from 'react';
+import './NavBar.css'
+
+export const NavBar: FunctionComponent<{categories: Array<string>}> = ({ categories }) => {
+    return <div>
+        {categories.map((category: any) => <><span className='movie-category'>{category}</span></>)}
```

Comment:
please add key attribute. It's crucial whenever we generate elements of the same type. Please read [this section in documentation](https://reactjs.org/docs/lists-and-keys.html#keys)